src/SourceIndexServer/Program.cs

```diff
@@ -7,15 +7,12 @@ public class Program
     {
         public static void Main(string[] args)
         {
-            var host = new WebHostBuilder()
-                .UseKestrel()
-                .UseContentRoot(Directory.GetCurrentDirectory())

```

Comment:
is this no longer needed? I think ContentRoot should still be needed so we can serve static content from the current directory?

Reply:
This is part of the new default WebHost builder. Reduces the boilerplate APIs you need to call. That's also why I could remove the ILoggerFactory.AddConsole() call.

https://github.com/aspnet/MetaPackages/blob/b40a1a5a8cfb186d41689f3c319135e518a7630c/src/Microsoft.AspNetCore/WebHost.cs#L152